src/Compilers/CSharp/Portable/Compilation/CSharpSemanticModel.cs

```diff
@@ -1852,7 +1852,7 @@ private bool IsInTypeofExpression(int position)
         private static void AddUnwrappingErrorTypes(ArrayBuilder<Symbol> builder, Symbol s)
         {
             var originalErrorSymbol = s.OriginalDefinition as ErrorTypeSymbol;
-            if ((object)originalErrorSymbol != null)
+            if ((object)originalErrorSymbol != null && originalErrorSymbol.CandidateSymbols != null)
```

Comment:
`ErrorTypeSymbol.CandidateSymbols` should return `ImmutableArray<Symbol>.Empty` rather than `default(ImmutableArray<Symbol>)`.